docs/content/policy-reference.md

```diff
@@ -824,6 +824,48 @@ Timezones can be specified as
 
 Note that the opa executable will need access to the timezone files in the environment it is running in (see the [Go time.LoadLocation()](https://golang.org/pkg/time/#LoadLocation) documentation for more information).
 
+Below is an example that uses some of the Time built-in functions.
+
+This package will check if a container image is older than 2 months. 
+
+**time_example.rego**
+```live:example/time:module:openable,merge_down
+package time_example
```

Comment:
I think this policy could probably be reduced to something like:

```rego
package time_example

import future.keywords.in

violation[image.name] {
    some image in input.images
    [year, month, _, _, _, _] := time.diff(time.parse_rfc3339_ns(image.created), time.now_ns())
    outdated(year, month)
}

outdated(year, month) {
    year > 0
}

outdated(year, month) {
    year == 0
    month > 2
}
```

Reply:
I'm wondering though if it wouldn't be better to have a policy demonstrate at least 5-6 of the built-in time functions. Right now the focus is more on the data than the actual time built-ins... WDYT?

Reply:
Thank You for review ! I will work on it.